setup.py

```diff
@@ -26,5 +26,5 @@ def read(fname):
     packages=['forecastio'],
     package_data={'forecastio': ['LICENSE.txt', 'README.rst']},
     long_description=open('README.rst').read(),
-    install_requires=['requests>=1.6', 'responses'],
+    install_requires=['requests>=1.6', 'responses<=0.23.1'],
```

Comment:
This is not correct. `responses` is for testing, it does not belong in this `install_requires` list at all.